shared/ff_util/include/ff_util/ff_names.h

```diff
@@ -382,6 +390,9 @@
 #define TOPIC_HARDWARE_IMU                          "hw/imu"
 #define TOPIC_HARDWARE_NAV_CAM                      "hw/cam_nav"
 #define TOPIC_HARDWARE_DOCK_CAM                     "hw/cam_dock"
+#define TOPIC_HARDWARE_CAM_SUFFIX_BAYER_RAW         "_bayer/image_raw"
+#define TOPIC_HARDWARE_CAM_SUFFIX_BAYER_MONO        "_bayer/image_mono"
```

Comment:
I have mixed feelings about including this topic suffix. Probably, nobody should ever subscribe to `/hw/cam_nav_bayer/image_mono` because its content is essentially the same as `/hw/cam_nav`, but subscribing to it would cause the `debayer` nodelet to redundantly do the same processing that `is_camera` already did.

(Actually, the two versions of debayering would be very subtly different due to the R/B channel swap issue, but that makes it even more of a confusing trap...)

If you really think we should have this here, I would recommend a comment that explains the issue.